doc/monitoring/index.rst

```diff
@@ -0,0 +1,441 @@
+.. _monitoring:
+
+===================
+Monitoring overview
+===================
+
+The aim of this part of the documentation is to explain the Ceph monitoring stack and the meaning of the main and basic Ceph metrics.
+
+With a good understand of the Ceph monitoring stack and metrics users can create customized monitoring tools, like prometheus queries, grafana dashboards, or scripts.
+
+
+Ceph Monitoring stack
+=====================
+
+Ceph provides a default monitoring stack wich is installed by default by cephadm and explained in :doc:`../cephadm/services/monitoring.rst` section.
+
+
+Ceph metrics:
+==============
+
+The main source for Ceph metrics are the performance counters exposed by each Ceph daemon. The :doc:`../dev/perf_counters` are native Ceph monitoring data
+
+Performance counters are transformed into standar Prometheus metrics using the Ceph exporter daemon. This daemon runs on every Ceph cluster host and exposes a metrics end point where all the performance counters exposed by all the Ceph daemons running in the host are published in the form of Prometheus metrics.
+
+Apart of the Ceph exporter, there is another agent to expose Ceph metrics. It is the Prometheus manager module, wich exposes metrics related to the whole cluster, basically metrics that are not produced by especific Ceph daemons.
```

Comment:
s/Apart of/In addition to/
s/especific/individual/

Reply:
Incorporated.